### DISCUSSION TITLE
Feature request: Is it possible to show the amount of annual heat losses as KPI

### DISCUSSION THREAD

**Discussion by RonaldBerk** [Project-OMOTES/user_feedback#59] [Q&A]
Knowing the ammount of annual heat losses is good for discussion and shows the effect of using (more expensive) twin-pipes and lower supply temperatures.

**Comment by ewoudwerkman** [comment]
As Kobus has indicated: this can be added as a KPI to the output of an optimizer run.

**Comment by jarsarasty** [comment]
Hi @RonaldBerk,

If I understood our discussion last week correctly, an approximation within 40% of the actual value of the losses would be sufficient for the conceptual design phase. We would expect the optimizer results to give an approximation within this range for this KPI. We should review the documentation related to the heat loss calculation.

**Comment by jarsarasty** [comment]
Initially could be computed for the whole system

**Comment by jarsarasty** [comment]
Hi @RonaldBerk ,

A [feature request](https://github.com/Project-OMOTES/user_feedback/issues/69) has been created.
